README.md

```diff
@@ -84,9 +84,14 @@ The LIDAR was used to gather all the data used to map the environment. The raw d
 
 ### Cartographer
 <p>
-Cartographer is an algorithm/system used to get a representation of the local environment given useful sensor data. For localization and mapping of the local environment, SLAM is the main part. 
+The Cartographer system is derived from Google’s repository in GitHub where the system is an open source project, letting us to use it. This system’s main focus is to be a solution to the simultaneous localization and mapping (SLAM) problem that is about how one can do both localization and mapping at the same time. 
+</p>
+
+<p>
+Cartographer is an algorithm used to get a representation of the local environment given useful sensor data. For localization and mapping of the local environment, SLAM solving is the main reason to use Cartographer. It contains two SLAMs, one local SLAM and one global SLAM. The local SLAM's function is to generate approved submaps with correct scans (range data) and the global SLAM combines all submaps and creates a global map. The Global map can then be used to visually represent the environment.
+
+The usefulness of this system is in the quality of representing the environment, high resolution maps can be easily created with this system. Cartographer itself creates a compressed protobuf file (pbstream) as an output which only contains what Cartographer claims to be useful sensor data and then when recombined with full sensor data file (bag) creating a high resolution map. Although it's in our interest to combine multiple maps from multiple robots into one map, there was no clear way to do that with the system. Instead the output file in interest became a portable grey map (PGM) file. For Cartographer to work with the rest of the parts, necessary tuning on the system’s parameters had to be done to work with the turtlebot3.  
```

Comment:
Change "high resolution" to "high-resolution".

Add "," after "Instead the output file..."